pkg/kubelet/images/image_gc_manager.go

```diff
@@ -50,6 +50,14 @@ const instrumentationScope = "k8s.io/kubernetes/pkg/kubelet/images"
 // indexed as imageId-RuntimeHandler
 const imageIndexTupleFormat = "%s,%s"
 
+// ImageGarbageCollectedTotalReason* specify the reason why
+// an image was garbage collected, to help operators decide if their
+// workloads are healthy as a result of enabling ImageGCMaxAge feature.
```

Comment:
I don't really understand this comment. Why would GC impact if workloads are healthy?

Reply:
the risk is thrashing with GC and a CronJob or something where an image is removed soon before it is set to be run. Documenting the reason will let an operator see if excessive image gc'ing is from this field or just happening generally.

Reply:
So this metric is only valid if ImageGCMaxAge is enabled? I believe space existed before your feature?

Reply:
right with the "reason" they'll differentiate the reason between "age" (added in this kep) and "space" (the old reason it used to happen, which still can happen now)

Reply:
My issue is that this isn't really that clear with just this standalone comment (if I didn't know anything about the KEP).

Reply:
I would just leave 
// ImageGarbageCollectedTotalReason* specify the reason why
// an image was garbage collected

Reply:
fair enough, updated